desktop/src/main/java/bisq/desktop/util/CssTheme.java

```diff
@@ -40,13 +44,18 @@ public static void loadSceneStyles(Scene scene, int cssTheme) {
         }
 
         scene.getStylesheets().setAll(
-            // load base styles first
-            cssThemeFolder + "bisq.css",
-            cssThemeFolder + "images.css",
-            cssThemeFolder + "CandleStickChart.css",
+                // load base styles first
```

Comment:
Indent ?

Reply:
It was badly indented before, should be 8 for arguments.

Reply:
sorry this was one of my early contributions and I had my text editor set to hard tabs or something